lib/board.ex

```diff
@@ -22,6 +22,20 @@ defmodule Board do
     end
   end
 
+  @spec move_status(map(), integer()) :: atom()
+  def move_status(board, player_move) do
```

Comment:
This is where the line gets blurry - does it belong in `Board` or `Validator`. What are your thoughts?

Reply:
I completely agree, the lines get blurry and that is a good question, after talking about it with Eli before, I think the best place for this validation is in the `Board` module, where it can then be brought into the `Validator` module. I think this is the best place because unlike the non numerical validation (which has no relation to the board, it is simply just input validation, which is why I thought it would be best in its own module), the out of bounds validation has to take into account how big the board is, so because of this relationship with the `Board`, I think the best place is in the `Board` module. What do you think? I appreciate all of the help and comments! 😄